nevergrad/parametrization/core.py

```diff
@@ -305,6 +305,17 @@ def set_name(self: P, name: str) -> P:
 
     # %% Constraint management
 
+    def constraint_penalties(self) -> tp.List[float]:
+        """Returns the list of constraint penalties penalties
+
+        Returns
+        -------
+        list of float
+            the list of penalties for each constraint
+        """
+        val = self.value
+        return sum(utils.float_penalty(func(val)) for func in self._constraint_checkers)
```

Comment:
isnt that a float and not a list of floats? I thing a list of floats is more general (and easy to sum, while we cannot unsum the float). Also, I expect this will soon disappear so for now we should probably keep it hidden

Reply:
Right! Fixed.